### DISCUSSION TITLE
Is there any pretrained paddle model for number only ?

### DISCUSSION THREAD

**Discussion by huanidz** [PaddlePaddle/PaddleOCR#7727] [Q&A] [answered]
I want to recognize a numbers from a image but i have to use 'en' or 'latin', etc., sometime it will recognize character 'l' instead of 1 and 'o' instead of 0, is there any model that trained for numbers only ?

**Comment by Evezerest** [comment, marked as answer]
we don't provide the model yet, you can use some post-process algorithm for now.
